src/Storefront/Resources/views/storefront/element/cms-element-image-gallery.html.twig

```diff
@@ -483,7 +524,12 @@
                                                                                         <div class="gallery-slider-item">
                                                                                             {% block element_image_gallery_inner_zoom_modal_slider_item_zoom_container %}
                                                                                                 <div class="image-zoom-container"
-                                                                                                     data-image-zoom="true">
+                                                                                                     {% if not (image.mimeType starts with 'video/') %}data-image-zoom="true"{% endif %}>
```

Comment:
Are you sure? Shouldn't the inline if statement for the data attribute have the same indentation as the class attribute above?

Reply:
Yes, I am. A deeper nesting level always consists of 4 spaces. This is 5. Imagine not having `<div` but having (I know, retro :smile:) `<marquee` there. Then the indent with your method would be 9 spaces. We can be depending on attribute indent.

IMO its better this way, so you don't mistake the div's content with it's attributes :smile:

Reply:
I do agree with you in case the if statement changes the **content** inside the div element. But this one is an attribute indent. It's inside the **opening div tag** to conditionally output the `data-image-zoom="true"` attribute. Same thing [here](https://github.com/shopware/shopware/blob/951f3306069315e3b37a0bfb80933fbf35de1492/src/Storefront/Resources/views/storefront/element/cms-element-image-gallery.html.twig#L138) with 5 spaces.

Reply:
Oh boy, we both meant the same the whole time and I was blind. That was quite a bad day for me, I thing.

You're absolutely right, don't mind my blindness :see_no_evil: